pkg/controller/pxc/backup.go

```diff
@@ -212,3 +212,22 @@ func (h *minHeap) Pop() interface{} {
 	*h = old[0 : n-1]
 	return x
 }
+
+func (r *ReconcilePerconaXtraDBCluster) deletePITR(cr *api.PerconaXtraDBCluster) error {
+	collectorDeployment := appsv1.Deployment{
+		TypeMeta: metav1.TypeMeta{
+			APIVersion: "apps/v1",
+			Kind:       "Deployment",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      deployment.GetBinlogCollectorDeploymentName(cr),
+			Namespace: cr.Namespace,
+		},
+	}
```

Comment:
I see same code in 3 other places.
Maybe it should be moved in separate func which returns collectorDeployment spec

Reply:
https://github.com/percona/percona-xtradb-cluster-operator/pull/704/commits/8a0bb3f16ef8c17b9c42c81e791473e977fc074d